jamal.go

```diff
@@ -0,0 +1,132 @@
+//usr/bin/env go run $0 $@; exit;
```

Comment:
Unix automatically normalizes paths. Go treats // as a single line comment. That is how you do a Golang shebang.